README.md

```diff
@@ -14,4 +14,4 @@ Copy everything you need from your previous solution.
 
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_list-of-goods-hooks/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<stunnerboyone>.github.io/react_list-of-goods-hooks/) and add it to the PR description.
```

Comment:
Fix your link in PR description. And fix 404 error, try to re-deploy it and check github link before re-request review

```suggestion
- Replace `<your_account>` with your Github username in the [DEMO LINK](https://stunnerboyone.github.io/react_list-of-goods-hooks/) and add it to the PR description.
```